securedrop/manage.py

```diff
@@ -18,8 +19,11 @@
 from db import db_session, init_db, Journalist
 from management.run import run
 
+logging.basicConfig(format='%(asctime)s %(levelname)s %(message)s')
+log = logging.getLogger(__name__)
 
-def reset():  # pragma: no cover
+
+def reset(args):  # pragma: no cover
```

Comment:
Ok, but then it should be added by either of those PR's, not this one. I say this for two reasons:

1. In general, the changes in a PR should be tightly scoped and limited to only the changes necessary to address the corresponding issue. This makes reviewing, `git blame`ing, etc. much easier.
2. Adding `args` in this PR will create merge conflicts in your other PR's. I don't see why you wouldn't just wait to add it in one of the other PR's instead (where it is actually needed/used by other changes).